pages/pigx-sars-cov2-ww.md

```diff
@@ -267,6 +271,31 @@ characterising variants of concern (VOC) of SARS-CoV-2 provided by
 [outbreak.info](https://outbreak.info/situation-reports) and
 [CoVariant.org](https://covariants.org/variants/S.501Y.V1).
 
+## Output Folder Structure
+
+The pipeline will create a specific directory structure in the, the respective contents are explained below:
+
+```
+|-- coverage
+|-- kraken
+|-- logs
+|-- mapped_reads
+|-- pigx_work
+|-- report
+|-- trimmed_reads
+|-- variants
```

Comment:
Since this is just a flat list I'd prefer to just, you know, make a flat list here instead of ASCII art.